src/pages/Documentation/index.tsx

```diff
@@ -198,10 +201,13 @@ export const Documentation: React.FC = (props) => {
             Participate as a bidder
           </IndexLink>
           <IndexLink activeClassName="isActive" to="/docs/participate-as-auctioneer#topAnchor">
-            Participate as auctioneer
+            Start an auction via cmd
```

Comment:
I renamed it for a shorter naming and more similar to the       
      Start an auction via UI

Participating wasn't the right word in my opinion